### DISCUSSION TITLE
请问怎么保存脚本？

### DISCUSSION THREAD

**Discussion by qiguangtao** [zyyzyykk/kkTerminal#2] [General]
请问怎么保存脚本？   想快捷录入保存的脚本

**Comment by zyyzyykk** [comment]
在Command Code里面输入 `SCCW` 并按下回车键, 可以添加自定义脚本，有一个部署jar包的默认模板

**Comment by zyyzyykk** [comment]
可以参考这个文档：
https://github.com/zyyzyykk/kkTerminal/blob/master/doc/zh_CN/CMDCODE.md
